.vscode-test.mjs

```diff
@@ -8,27 +8,34 @@ import fs from 'fs';
 import path from 'path';
 
 // Discover test suite folders in src/test/integration
-const BASE_SRC_PATH = './src/test/integration';
+const BASE_SRC_PATH = './src/test/fixtures';
 const BASE_OUT_PATH = './out/test/integration';
```

Comment:
It seems like this directory does not exist anymore, so no tests would run? If they still did, maybe you had stale code in `out/`?